console/src/components/Main/Actions.js

```diff
@@ -273,6 +279,8 @@ const mainReducer = (state = defaultState, action) => {
       return { ...state, loginInProgress: action.data };
     case LOGIN_ERROR:
       return { ...state, loginError: action.data };
+    case RUN_TIME_ERROR: // To trigger telemetry event
+      return { ...state };
```

Comment:
Does it not have to be copy of state? If this is fine please feel free to commit.

Reply:
If we made a copy then everything relaying on this state would be refreshed, because React would notice that this is a different object. Here we want to only handle this case and say the same as we do for `default` case:  `hey React, don't do anything, the state was not changed, it's the same`.